### PR TITLE
Day 5

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/components/ProjectHeader.js [hunk 1]
```diff
@@ -5,28 +5,79 @@ import {
...
 
+  const [isOpen, setIsOpen] = useState([false, false]);
+  const onToggle = (index) =>
+    setIsOpen((prevIsOpen) => {
+      const newIsOpen = [false, false];
+      newIsOpen[index] = !prevIsOpen[index];
+      return newIsOpen;
+    });
+
+  const navMenuItems = [
```
Comment (aannyyaa) [inline src/components/ProjectHeader.js:31]: Right now, I have both a drop down menu for language selection and standalone buttons - I'll end up choosing one or the other!

File: src/styles/app.scss [hunk 2]
```diff
@@ -1,9 +1,18 @@
+$mainBackgroundColor: #e1f3f8;
+// @import './settings.scss';
+
 main {
-  padding-bottom: 200px;
-  background-color: #e1f3f8;
+  padding-bottom: 250px;
+  // background-color: $theme-color-primary-light;
```
Comment (aannyyaa) [inline src/styles/app.scss]: For some reason using the $theme-color-primary-light variable doesn't seem to work for me (even when I import settings.scss on line 2). I realized that I haven't been compiling my scss files. Do I need to be doing so in a create-react-app?

File: src/components/ProjectHeader.js (hunk 1 above)
Comment (saifulislamdev) [reply src/components/ProjectHeader.js:31]: Preference to choose the drop-down menu.

File: src/styles/app.scss (hunk 2 above)
Comment (saifulislamdev) [reply src/styles/app.scss]: I have also been stuck on SASS styling, as indicated in my last PR, so I won't be of much help.

File: src/styles/app.scss [hunk 3]
```diff
@@ -1,9 +1,12 @@
+$mainBackgroundColor: #e1f3f8;
+// @import './settings.scss';
+
 main {
-  padding-bottom: 200px;
-  background-color: #e1f3f8;
+  padding-bottom: 250px;
```
Comment (saifulislamdev) [inline src/styles/app.scss:4]: Make sure your padding is good for all screen orientations and sizes.

File: src/App.js [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
-  Route,
-  Switch,
-  useHistory,
-} from "react-router-dom";
+import { Route, Switch, useHistory, useLocation } from "react-router-dom";
 import "@trussworks/react-uswds/lib/uswds.css";
 import "@trussworks/react-uswds/lib/index.css";
 import { useTranslation } from "react-i18next";
-import { Grid} from "@trussworks/react-uswds";
+import { Grid } from "@trussworks/react-uswds";
```
Comment (saifulislamdev) [inline src/App.js:6]: Hehe.

File: src/components/TeamCard.js [hunk 5]
```diff
@@ -18,12 +22,8 @@ const Team = ({ name, info, photo, role, year }) => {
...
       <CardBody>{info}</CardBody>
       <CardFooter>
-        <Tag>
-          {role}
-        </Tag>
-        <Tag>
-          {year}
-        </Tag>
+        <Tag>{role}</Tag>
+        <Tag>{year}</Tag>
```
Comment (saifulislamdev) [inline src/components/TeamCard.js:26]: Interesting.

File: src/styles/app.scss [hunk 6]
```diff
@@ -9,4 +9,12 @@ main {
 
 #footer {
   margin-top: -200px;
-}
\ No newline at end of file
+}
+
+#NavMenuButtonContainer {
```
Comment (saifulislamdev) [inline src/styles/app.scss:12]: I did almost the same thing for my language switcher.

File: src/App.js [hunk 7]
```diff
@@ -25,6 +25,10 @@ const App = () => {
 
   let currentLanguage = i18n.language;
 
+  useEffect(() => {
+    document.title = t("docTitle");
+  }, [currentLanguage, t]);
```
Comment (saifulislamdev) [inline src/App.js]: Why is `t` a dependency as well? I might be missing something but it seems like you don't need it. If you do need it, then maybe you might not need `currentLanguage` too. Also, I forgot where I saw it, but each view should have its own page title (maybe a11y checklist or some other accessibility tool).

File: src/styles/app.scss (hunk 3 above)
Comment (aannyyaa) [reply src/styles/app.scss:4]: Thanks - I was using padding so the footer have an absolute position at the bottom but not be covered by or cover the main content. Maybe I could just use Grid?

File: src/styles/app.scss (hunk 2 above)
Comment (aannyyaa) [reply src/styles/app.scss]: No problem. If I find any helpful sass tutorials/resources I'll share them!

File: src/App.js (hunk 7 above)
Comment (aannyyaa) [reply src/App.js]: Thanks for pointing out that each view should have its own title.

And, good question! I originally didn't have 't' as a dependency, but then received an es-lint warning: "React Hook useEffect has a missing dependency: 't'. Either include it or remove the dependency array." I tried taking 't' out and everything still seems to work. I only want the useEffect hook to run when the language changes, so it seems like taking it out should be okay

File: src/styles/app.scss (hunk 3 above)
Comment (saifulislamdev) [reply src/styles/app.scss:4]: Try it! If you remember, let me know how it turns out.

Comment (saifulislamdev) [reply src/styles/app.scss:4]: Try it! If you remember to, let me know how it turns out!

**Pull request merged**